config/routes.rb

```diff
@@ -15,4 +15,7 @@
   resources :users, only: [:none] do
     resources :interactions, except: [:show]
   end
+
+  post 'twilio/answer' => 'twilio#answer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.